srcpkgs/spiped/template

```diff
@@ -1,19 +1,21 @@
 # Template file for 'spiped'
 pkgname=spiped
 version=1.6.1
-revision=2
+revision=3
 build_style=gnu-makefile
 makedepends="openssl-devel"
 short_desc="Spiped secure pipe daemon"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Anthony Iliopoulos <ailiop@altatus.com>"
 license="BSD-2-Clause"
 homepage="http://www.tarsnap.com/spiped.html"
 distfiles="http://www.tarsnap.com/spiped/spiped-${version}.tgz"
 checksum=8d7089979db79a531a0ecc507b113ac6f2cf5f19305571eff1d3413e0ab33713
+system_accounts="_spiped"
```

Comment:
Shouldn't this be passed in configure somewhere as well? Or is it just so someone doesn't have to create it? Not sure I understand the purpose.

Reply:
There are no configure options as by default spiped runs with the current user privileges, unless the -u option is passed.

It's so that an account doesn't have to be created manually and so that we have "standardized" sandbox accounts as much as possible. 

Ideally this would be accompanied by a sv script, but given that a tunnel can be created for any arbitrary service/port I wasn't sure if I should provide one. Would it make sense to provide a run script that needs a related conf file (for configuring the various options)? Also, that would cover a single tunnel instance only, anyone that needs more would need to create more services (e.g. /etc/sv/spiped-tunnel-N etc.).

Reply:
Why don't it use `nobody`?

Reply:
I'd like to avoid doing that, given that `nobody` it's a catch-all account and maybe shared by different services. There are far too many ways that processes running with the same creds can influence each other, and it's better to avoid that as much as possible given that this is done for increased security, especially for long-running daemons. Plus there are further things that can be restricted by uid so it's useful to have separation.

I suppose I'd better provide a parameterized sv file, similar to what the freebsd pkg does [1], and let users configure the various options over a conf file as we do for the rest (e.g. like sshd). This would also have the system accounts hardcoded, so it would make more sense.

[1] https://cgit.freebsd.org/ports/tree/sysutils/spiped/files/spiped.in